### PR TITLE
[SELS-TASK] Integration of Quizpage

### PR DESCRIPTION
**Pull request by NiloJr-sun**
## Asana Link 
- [Integration of Quizpage](https://app.asana.com/0/1202703680910766/1202861165457834/f)
## Commands
```
-cd backend
- php artisan serve
- cd frontend
- npm start
```
##  Difinition of done
- [x] Integration of Quizpage

## Notes
- 

## Screenshots
image
image

### CODE REVIEW

File: backend/app/Http/Controllers/CategoryController.php
Comment (johnpaul-sun) [inline backend/app/Http/Controllers/CategoryController.php]: Maybe you can do this if you're getting only one data via ID from Category
`
 $category = Category::findOrFail($id);
`

File: frontend/src/components/userdashboard/quizPage.js
Comment (johnpaul-sun) [inline frontend/src/components/userdashboard/quizPage.js]: You can insert all functions inside of one useEffect if the dependencies are none for all.
If you're going to depend on some state for future purposes, please include it using comment.

```
 useEffect(() => {
        setAns('');
        fetchTitle(categoryId);
        fetchWord(categoryId);
    }, []) 
```

File: frontend/src/components/userdashboard/quizPage.js
Comment (johnpaul-sun) [inline frontend/src/components/userdashboard/quizPage.js]: remove the console log

**Review by johnpaul-sun — APPROVED** [review]
I approved these changes, please wait for Sir EJ for his review and approval for this PR to be merged.

**Review by ejanton-sun — APPROVED** [review]
# Approve

**Pull request merged**
